test/servico/Cst20Test.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import dominio.Usuario;
+import java.util.List;
+import modelo.UsuarioModel;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Victor Messina
+ */
+public class Cst20Test {
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+    }
+
+    @Test
+    public void testeCriarUsuario() {
+        LoginService loginService = new LoginService();
+        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
+
+        AdministracaoDeUsuariosService aus;
+
+        aus = new AdministracaoDeUsuariosService();
+
+        Usuario usuario1 = new Usuario();
+
+        usuario1.setUsuario("ProfessorTest");
+        usuario1.setSenha("123456");
+        usuario1.setPermissao(Perfil.PROFESSOR);
+
+        List<UsuarioModel> usuariosCadastrados;
+        usuariosCadastrados = aus.ListarUsuarios();
+
+        for (UsuarioModel um : usuariosCadastrados) {
+            if (!um.Usuario.equals(usuario1.getUsuario())) {
+                aus.SalvarUsuario(usuario1.getUsuario(), usuario1.getSenha(), usuario1.getPermissao());

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 103).](https://www.codacy.com/app/calebepb/substituicao/file/2734087542/issues/source?bid=3308597&fileBranchId=3351034#l49)